scss/civicrm/common/_base.scss

```diff
@@ -26,6 +26,10 @@ body {
     }
   }
 
+  .status-overdue {
```

Comment:
don't we have all the `.status-` related style all bundled together somewhere? This here seems a bit thrown in without  any structure in mind

Reply:
No we just have 2 of such style 
another is
as per below screenshot

![image](https://user-images.githubusercontent.com/26058635/37215517-2476e052-23de-11e8-8cb6-aa66ded22c07.png)

"Past Groups" is `h3` with `.status-removed` and other one is `td`.

Should I create a separate partial for text (options other than `h3`)?